FlightPlanner/FlightPlanner.Core/Search/ISearch.cs

```diff
@@ -0,0 +1,10 @@
+﻿using FlightPlanner.Core.Models;
+
+namespace FlightPlanner.Core.Search
+{
+    public interface ISearch
```

Comment:
te atkal esi kontekstu ievilkusi implementācijā. :) labāk tad airport service uztaisi metodi search Airports.. un flight service uztaisi metodi search flights. :) un tas izskatīsies tīrāk. :) un tad ja gribi to visu apvienot zem viena tad implementācijā lieto servisus? :)